pom.xml

```diff
@@ -105,11 +105,161 @@
         <version>2.5</version>
       </plugin>
     </plugins>
+    <pluginManagement>
+      <plugins>
+        <plugin>
+          <groupId>io.jenkins.tools.incrementals</groupId>
+          <artifactId>incrementals-maven-plugin</artifactId>
+          <version>${incrementals-plugin.version}</version>
+          <configuration>
+            <includes>
+              <include>org.jenkins-ci.*</include>
+              <include>io.jenkins.*</include>
+            </includes>
+            <generateBackupPoms>false</generateBackupPoms>
+            <updateNonincremental>false</updateNonincremental>
+          </configuration>
+        </plugin>
+      </plugins>
+    </pluginManagement>
   </build>
 
   <properties>
+    <revision>build-217-jenkins-25</revision>
+    <changelist>-SNAPSHOT</changelist>
+    <scmTag>HEAD</scmTag>
+    <incrementals-enforce-minimum.version>1.1</incrementals-enforce-minimum.version>
+    <incrementals-plugin.version>1.1</incrementals-plugin.version>
+    <incrementals.url>https://repo.jenkins-ci.org/incrementals/</incrementals.url>
     <maven.compiler.source>1.8</maven.compiler.source>
     <maven.compiler.target>1.8</maven.compiler.target>
   </properties>
 
+  <profiles>
```

Comment:
Ouch a shame to have to use this here, could you inherit from the Jenkins parent Pom instead jenkinsci/pom?

Reply:
this is a library, not a plugin or a module, I've followed the instructions at https://github.com/jenkinsci/incrementals-tools#usage-in-other-poms and clone the changes on the PR https://github.com/jenkinsci/groovy-sandbox/commit/22d517234ce7d16c249bcb16dd59136f04ea10c5

Reply:
I am checking if I can move to use org.jenkins-ci:jenkins:1.57 as a parent, this library has a baned dependency.